pymongo/monitor.py

```diff
@@ -300,6 +300,7 @@ def __init__(self, topology, topology_settings):
         self._seedlist = self._settings._seeds
         self._fqdn = self._settings.fqdn
         self._srv_service_name = self._settings._srv_service_name
+        self._srv_max_hosts = self._settings._srv_max_hosts
```

Comment:
We no longer need to pass srv_max_hosts here do we?

Reply:
You're right, done.